src/AddressLabel.jsx

```diff
@@ -0,0 +1,78 @@
+const React = require('react');
+const { ReactiveComponent } = require('oo7-react');
+const { Label, Button } = require('semantic-ui-react');
+
+class AddressLabel extends ReactiveComponent {
+	constructor () {
+		super(['address']);
+		this.state = {
+			isCopyHovered: false
+		};
+	}
+
+	onCopyAddress (text) {
+		var textArea = document.createElement('textarea');
```

Comment:
No, this is forbidden in React. You cannot do DOM manip with React. I'm sure there's a way to do the same without DOM manipulation

Reply:
fixed